shared/message/motion/KinematicsModel.proto

```diff
@@ -108,10 +136,12 @@ message KinematicsModel {
         mat3 foot        = 9;
     }
 
-    Leg         leg                       = 1;
-    Head        head                      = 2;
-    Arm         arm                       = 3;
-    MassModel   massModel                 = 4;
-    TensorModel tensorModel               = 5;
-    float       TEAMDARWINCHEST_TO_ORIGIN = 6;
+    // Instantiations of the config containers.
+    Leg         leg         = 1;
+    Head        head        = 2;
+    Arm         arm         = 3;
+    MassModel   massModel   = 4;
+    TensorModel tensorModel = 5;
+
+    float TEAMDARWINCHEST_TO_ORIGIN = 6;
```

Comment:
Epic hack from the days of the darwin when we used team darwins walk engine

Reply:
Would it cause problems if removed?

Reply:
Doubt it would cause problems anymore